packit_service/worker/events/event.py

```diff
@@ -379,29 +381,42 @@ def get_package_config(self) -> Optional[PackageConfig]:
             package_config.upstream_project_url = self.project_url
         return package_config
 
+    @staticmethod
+    def _filter_failed_models_target(
+        models: Union[
+            Optional[Iterable[CoprBuildModel]], Optional[Iterable[TFTTestRunModel]]
+        ],
+    ) -> Optional[Set[str]]:
+        failed_models_target = set()
+        for model in models:
+            if model.status in ["failed", "error"]:
+                failed_models_target.add(model.target)
+
+        return failed_models_target if failed_models_target else None
```

Comment:
```suggestion
    @staticmethod
    def _filter_failed_models_targets(
        models: Union[
            Optional[Iterable[CoprBuildModel]], Optional[Iterable[TFTTestRunModel]]
        ],
    ) -> Optional[Set[str]]:
        failed_models_targets = set()
        for model in models:
            if model.status in ["failed", "error"]:
                failed_models_targets.add(model.target)

        return failed_models_targets if failed_models_targets else None
```

Just to make it explicit that it's an iterable. (Btw. do you want to use `None` instead of an empty set?)